vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/holiday_access_rule.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module SecureRooms
+
+  module AccessRules
+
+    class HolidayAccessRule < BaseRule
```

Comment:
I thought that we would need to reference this class from the `CheckAccess` list of rules, no?